zookeeper.py

```diff
@@ -5,42 +5,55 @@
 # Copyright (c) 2016, InnoGames GmbH
 #
 
-import time
-import socket
-from socket import gethostname
+from argparse import ArgumentParser
+from socket import socket
```

Comment:
In the lines 45-48 `socket.AF_INET`, `socket.SOCK_STREAM` and `socket.SHUT_WR` are used. You have to import then too.